### PR TITLE
Changes for Moderate Enhanced projects

### PR DESCRIPTION
**Pull request by isubsmith**
Adds "citadel" host key.
Adds batch-spi and batch-spi-hm queues to Summit user guide
Adds lots of stuff about file systems. Clarifies when projects get HPSS storage as well (e.g. moderate export control does not have world shared or HPSS)

### CODE REVIEW

**Comment by isubsmith** [comment]
Don't merge this until we get the approval from Robert Michael

**Review by chrisfuson — APPROVED** [review]

**Pull request merged**
